apis/config/v1beta3/types.go

```diff
@@ -143,7 +143,10 @@ type ScoringStrategy struct {
 type NodeResourceTopologyMatchArgs struct {
 	metav1.TypeMeta `json:",inline"`
 
+	// ScoringStrategy a scoring model that determine how the plugin will score the nodes.
 	ScoringStrategy *ScoringStrategy `json:"scoringStrategy,omitempty"`
+	// If > 0, enables the caching facilities of the reserve plugin - which must be enabled
+	CacheResyncPeriodSeconds *int64 `json:"cacheResyncPeriodSeconds,omitempty"`
```

Comment:
It will be used in followup to implement the reconciliation logic: https://github.com/kubernetes-sigs/scheduler-plugins/pull/437/commits/c5f65867bdd22933213024e334a4ca35b4d8f804#diff-4ad175c1f91ec022ebcea71714a329e01f594c0ac6487650b3d779eb33e08f41R164